appeals/web/src/server/lib/mappers/data/appellant-case/submappers/application-decision.js

```diff
@@ -0,0 +1,23 @@
+import { textSummaryListItem } from '#lib/mappers/components/index.js';
+import { APPEAL_APPLICATION_DECISION } from 'pins-data-model';
+
+/** @type {import('../mapper.js').SubMapper} */
+export const mapApplicationDecision = ({ appellantCaseData, currentRoute, userHasUpdateCase }) =>
+	textSummaryListItem({
+		id: 'application-decision',
+		text: 'Outcome',
+		value: (() => {
```

Comment:
Would it be more consistent to define this as a named function in this same file, instead of inline, as with `mapOwnersKnownLabelText`?

Reply:
Personally I love a selfcalling function but yeah I agree it's more in line with the other funcs that we pull this out